distribution/docs/src/main/resources/content/_architectures/content-directory-monitor.adoc

```diff
@@ -61,23 +61,31 @@ For local files, the default value of 500 milliseconds is recommended. The recom
 It is also recommended that the *ReadLock Time Interval* be set to a lower amount of time when the *Maximum Concurrent Files* is set above 1 so that files are
 locked in a timely manner and processed as soon as possible.  When a higher *ReadLock Time Interval* is set, the time it takes for files to be processed is increased.
 
-.Attribute Mapper
-The CDM supports setting metacard attributes directly when ${branding} ingests a file. Custom mappings are entered in the form:
+.Attribute Overrides
+The CDM supports setting metacard attributes directly when ${branding} ingests a file. Custom overrides are entered in the form:
 
 `*attribute-name=attribute-value*`
 
-For example, to set the contact email for all metacards, add the custom mapping:
+For example, to set the contact email for all metacards, add the attribute override:
 
 `*contact.point-of-contact-email=doctor@clinic.com*`
 
-Each mapping sets the value of a single metacard attribute. To set the value of an additional attribute, select the "plus"
+Each override sets the value of a single metacard attribute. To set the value of an additional attribute, select the "plus"
  icon in the UI. This creates an empty line for the entry.
 
-To set multi-valued attributes, use a separate override for each value. For example, to add the keywords _PPI_ and _radiology_ to each metacard, add the custom attribute mappings:
+To set multi-valued attributes, use a separate override for each value. For example, to add the keywords _PPI_ and _radiology_ to each metacard, add the custom attribute overrides:
 
 `*topic.keyword=PPI*` +
 `*topic.keyword=radiology*`
 
+Attributes will only be overridden if they are part of the <<_metacard_type, metacard type>>, however the attributes do not have to present on the metacard to be set.
```

Comment:
❓ Perhaps we should link to the attribute injection documentation here?

Reply:
Could you link the section you're thinking of?

Reply:
I was thinking of https://github.com/codice/ddf/blob/50ff030c1c660b73c8bbd223c9cd9b2f02efad4b/distribution/docs/src/main/resources/content/_devComponents/attribute-injection.adoc

Reply:
So we could say something like "To override attributes that are not part of the metacard type, attribute injection can be used to add the attribute to the metacard...(link to section)"?

Reply:
Exactly, or just make `attribute injection` the link. I'll defer to @ricklarsen on our style preference there.

Reply:
I'd probably go with `... <<_attribute_injection,attribute injection>>` can be used...`

Reply:
This works for me.